sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector_node.cpp

```diff
@@ -35,6 +35,8 @@ DistortionCorrectorComponent::DistortionCorrectorComponent(const rclcpp::NodeOpt
   base_frame_ = declare_parameter<std::string>("base_frame");
   use_imu_ = declare_parameter<bool>("use_imu");
   use_3d_distortion_correction_ = declare_parameter<bool>("use_3d_distortion_correction");
+  auto has_static_tf_only =
+    declare_parameter<bool>("has_static_tf_only", false);  // TODO(amadeuszsz): remove default value
```

Comment:
Only this occurrence has a comment. Is this on purpose or should it be added to all other instances?

Reply:
Other occurrences are placed in nodes which are not refactored yet, therefore default value for parameter is valid.

Reply:
Thanks !